node/time_heap.js

```diff
@@ -77,6 +77,7 @@ function TimeHeap(options) {
     self.lastTime = self.timers.now();
     self.timer = null;
     self.end = 0;
+    self.lastRun = 0;
```

Comment:
null is a more sensible default value since it is unlikely that this would have run in the 70’s.